tapir/accounts/templates/accounts/email/create_account_reminder.body.html

```diff
@@ -18,8 +18,8 @@
             or visit <a href="https://www.google.com/maps/place/SuperCoop+Berlin/@52.553371,13.3585897,15z/data=!4m5!3m4!1s0x0:0x72e80db955fa8d3!8m2!3d52.5533091!4d13.3586012">SuperCoop</a>.
             Feel free to drop by and meet other SuperCoopies in person! Opening hours are::
         <ul>
-            <li>Wednesday - Friday from 16:30 - 19:30 Uhr</li>
-            <li>Saturdays from 11.00 - 14.00 Uhr</li>
+            <li>Monday - Friday from 4:30 - 7:30 p.m.</li>
+            <li>Saturdays from 11 a.m. - 2 p.m.</li>
```

Comment:
Did Vicky specifically request a.m./p.m. notation? In my opinion it's the worst way to write time, and I think it's only used in north america.
Also, if we say "from" we should also say "to".
```suggestion
            <li>Monday - Friday from 16:30 to 19:30</li>
            <li>Saturdays from 11:00 to 14:00</li>
```

Reply:
Yes she did: [https://supercoopberlin.slack.com/archives/D056MLQJ0JY/p1727701347428449?thread_ts=1727205093.752209&cid=D056MLQJ0JY](url) and am/pm is also in Britain the standard. Ok to let like this? I've changed to hyphen to "to". Not sure anymore: is it ok just to commit only the `create_account_reminder.body.html`? Are the servers doing the `makemessages` command?

Reply:
OK uses both, with 24h used more often. I would really never used am/pm anywhere, it's just bad.
The linked you copied doesn't work.